### PR TITLE
Sensu core supported platforms include Solaris and AIX

### LINKED ISSUE #871: Sensu Core supported platform matrix lacks entries for Solaris, AIX
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)

## Expected Behavior

Supported platform matrix in Sensu Core FAQ should be comprehensive

## Current Behavior

Supported platform matrix omits Solaris and AIX platform/architecture varieties.

## Possible Solution

Update matrix to include these platform/architecture varieties, and any others from 
https://sensu.io/products/downloads which are not already in the FAQ

### PR DESCRIPTION
**Pull request by rgeniesse**
## Description
Closes #871

### CODE REVIEW

**Comment by cwjohnston** [comment]
@amdprophet can you please help us provide accurate CPU architecture descriptions here? I can't recall the specifics myself.

**Comment by amdprophet** [comment]
Solaris 10 -x86, sparc
Solaris 11 - x86
AIX - powerpc

**Comment by cwjohnston** [comment]
> I think it’s all 32-bit
> sparcv9 (which actually is sparc64 I guess)

> but it’s generally only referred to as sparc64 by non-solaris projects, saying sparc should be fine
>as for the x86 builds, let me see
>it should be 32-bit for solaris 10 & 11, as the filenames contain i386

**Review by cwjohnston — CHANGES_REQUESTED** [review]
Please see Justin's comments in PR discussion

**Comment by rgeniesse** [comment]
@cwjohnston I _think_ I've added the correct entries.

**Comment by amdprophet** [comment]
We should have a third and forth column here for sparc and powerpc, solaris 10 and solaris 11 are 32-bit packages (not 64-bit), aix is powerpc.

File: content/sensu-core/1.6/faq.md [hunk 1]
```diff
@@ -15,24 +15,27 @@ Enterprise](https://sensu.io/features/enterprise) a try.
...
+| Debian 8           | ✅     | ❌     | x86     | No official 32bit images |
+| Debian 9           | ✅     | ❌     | x86     | No official 32bit images |
+| CentOS 5           | ✅     | ✅     | x86     | 32 and 64bit images built with [sensu-omnibus-packer](https://github.com/sensu/sensu-omnibus-packer) |
+| CentOS 6           | ✅     | ✅     | x86     | Using unofficial 32bit image |
+| CentOS 7           | ✅     | ❌     | x86     | No official 32bit images |
+| FreeBSD 10         | ✅     | ❌     | x86     | Official 32bit images are out of date |
+| FreeBSD 11         | ✅     | ❌     | x86     | No official 32bit images |
+| Solaris 10         | ❌     | ✅     | sparc   | No official 64bit images |
+| Solaris 11         | ❌     | ✅     | sparc   | No official 64bit images |
+| IBM AIX 6.1 +      | ❌     | ✅     | powerpc | No official 64bit images |
```
Comment (apaskulin) [inline content/sensu-core/1.6/faq.md]: Please remove the comments from lines 32-34

Comment (rgeniesse) [reply content/sensu-core/1.6/faq.md]: Done!

**Review by apaskulin — APPROVED** [review]
+1 to copy to all versions

**Pull request merged**